infra.bs

```diff
@@ -226,6 +228,60 @@ as well as the single algorithm would. Therefore performance is best left as a f
 over.
 
 
+<h3 id=declaration>Declaration</h3>
+
+<p>Algorithm names are usually verb phrases, but sometimes are given names that emphasize their
+standalone existence, so that standards and readers can refer to the algorithm more idiomatically.
+
+<p class=example id=example-algorithm-declaration-names>Some algorithm names in the latter category
+include "attribute change steps", "internal module script graph fetching procedure", and "overload
+resolution algorithm".
+
+Declare algorithms by stating their name, parameters, and return type, in the following form:
+
+<blockquote>
+To <dfn ignore>[algorithm name]</dfn>, given a [type1] <var ignore>[parameter1]</var>, a [type2]
+<var ignore>[parameter2]</var>, &hellip;, perform the following steps. They return a [return type].
```

Comment:
What's the ellipsis for?

Reply:
To indicate that you're not limited to only two parameters.